V1.0.0

mariohov opened the pull request:
Implementation of a data integration app.

ebbauer commented on README.md:
````diff
@@ -1 +1,53 @@
+
+```$ sudo docker run --rm -it -v "$(pwd)":/data --name neowl-mongo -p 127.0.0.1:27017:27017 -d mariocmp/neowl-mongo:1.0.0 mongod --smallfiles```
+
+### Pull and Run Neowl-go Docker Image
````
This title is the same as before. Could you point out the differences?

ebbauer replied:
ops my mistake

ebbauer commented:
Hi Mario.

I'm Eduardo Bauer and I'm reviewing your code.

Here are some notes that I found:
- I wasn't able to merge companies between q1 and q2 because of a 400 error using the swagger doc. Could you take a quick look at it?
- a problem at Makefile. 
make setup
Makefile:6: *** multiple target patterns.  Stop.

As for the rest, it is great. Good test cover, great documentation, easy to execute and nice language choosing :D

mariohov commented:
Hi Eduardo,

Did you use docker images or you built the project?
Let me try to reproduce the error.

ebbauer commented:
Thanks. I used mongo docker image and ran the project locally

mariohov commented:
@ebbauer 

I fixed Makefile and I found why merge function was sending a 400 error. The company's name should match exactly with the name in the database. It's not case insensitive, so I updated swagger example.
The merge result should be uppercase company's name and updated website data.
Could you please try again and let me know if it works?
Thanks.
